src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react"
+import { deleteComment } from "../../api"
+
+function DeleteComment({comment}) {
+    const [isDeleted, setIsDeleted] = useState(false)
+    const [isNotDeleted, setIsNotDeleted] = useState(false)
+    const [isDeleting, setIsDeleting] =useState(false)
+
+    //// Temp Logged in User ////
+    const username = "cooljmessy"
+
+    function handleClick() {
+        setIsDeleting(true)
```

Comment:
I would also consider how you might be able to optimistically remove the comment once it's been deleted for a nicer user experience (although not necessary at all!)
If you ever go down this route, remember it would need to be undone if the deletion in the db fails!